internal/logger/logger.go

```diff
@@ -0,0 +1,72 @@
+package logger
+
+import (
+	"encoding/json"
+	"log"
+	"os"
+
+	"go.uber.org/zap"
+)
+
+var (
+	envLog = os.Getenv("OPCAP_LOG_LEVEL")
+	cfg    zap.Config
+)
+
+type LogJSON struct {
+	EncoderConfig *EncoderCfg `json:"encoderConfig"`
+	Level         string      `json:"level"`
+	Encoding      string      `json:"encoding"`
+	OutputPaths   []string    `json:"outputPaths"`
+}
+
+type EncoderCfg struct {
+	MessageKey   string `json:"messageKey"`
+	LevelKey     string `json:"levelKey"`
+	LevelEncoder string `json:"levelEncoder"`
+}
+
+var logger *zap.Logger
+var sugarLogger *zap.SugaredLogger
+
+func InitLogger() {
+	data := &LogJSON{
+		Level:       envLog,
+		Encoding:    "json",
+		OutputPaths: []string{"stdout"},
+		EncoderConfig: &EncoderCfg{
+			MessageKey:   "message",
+			LevelKey:     "level",
+			LevelEncoder: "lowercase",
+		},
+	}
+	dataJSON, err := json.Marshal(data)
+	if err != nil {
+		log.Fatalf("failed to build zap logger: %v", err)
+	}
+
+	if err := json.Unmarshal([]byte(dataJSON), &cfg); err != nil {
+		panic(err)
+	}
+	logger, err = cfg.Build()
+	if err != nil {
+		panic(err)
+	}
+	sugarLogger = logger.Sugar()
+}
+
+func Info(message string, fields ...interface{}) {
+	sugarLogger.Info(message, fields)
+}
+func Debug(message string, fields ...interface{}) {
+	sugarLogger.Debug(message, fields)
+}
+func Error(message string, fields ...interface{}) {
```

Comment:
Opinion: These should remain as Errorf, Debugf, etc. as that is a convention when a format string is used, and a variable number of args for the format.

Reply:
I agree with your suggestion, doing it. Thank you.